modules/update-conditional-web-console.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-web-console.adoc
+
+:_content-type: PROCEDURE
+[id="update-conditional-web-console{context}"]
+= Performing conditional updates in the web console
+
+You can view and assess the risks associated with particular updates with conditional updates.
+
+.Prerequisites
+* You have access to the cluster with `cluster-admin` privileges.
+
+* You have access to the OpenShift Container Platform web console.
+
+* Your Operators previously installed through Operator Lifecycle Manager (OLM) are updated to their latest version in their latest channel. Updating the Operators ensures they have a valid update path when the default OperatorHub catalogs switch from the current minor version to the next during a cluster update.
+
+* Your machine config pools (MCPs) are running and not paused. Nodes associated with a paused MCP are skipped during the update process. You can pause the MCPs if you are performing a canary rollout update strategy.
+
+* Your {op-system-base}7 workers are replaced with {op-system-base}8 or {op-system} workers before updating to {product-title} {product-version}. Red Hat does not support in-place {op-system-base}7 to {op-system-base}8 updates for {op-system-base} workers; those hosts must be replaced with a clean operating system install.
```

Comment:
These content looks disconnected from what we should be communicating. @opayne1 did anyone suggested you the content. Else we can help with the rewrite of the content. cc @wking

Reply:
Thanks for taking a look @LalatenduMohanty! This content actually already existed, I just moved it around. We can always move it around or remove it if it is not relevant to this doc.

Reply:
Removed from this module. Thanks!